File: lib/model/notification_admin.dart
class NotificationAdmin {
  String? name;
  String? title;
  String? subtitle;
  String? time;
  String? date;
  String? avt;

  NotificationAdmin(
      {this.name, this.title, this.subtitle, this.time, this.date, this.avt});

  NotificationAdmin.fromJson(Map<String, dynamic> json) {
    name = json['name'];
    title = json['title'];
    subtitle = json['subtitle'];
    time = json['time'];
    date = json['date'];
    avt = json['avt'];
  }

  Map<String, dynamic> toJson() {
    final Map<String, dynamic> data = new Map<String, dynamic>();
    data['name'] = this.name;
    data['title'] = this.title;
    data['subtitle'] = this.subtitle;
    data['time'] = this.time;
    data['date'] = this.date;
    data['avt'] = this.avt;
    return data;
  }
}

File: services/model/notification_admin.js
const mongoose = require("mongoose");
const notificationAdminSchema = mongoose.Schema({
  name: {
    require: true,
    type: String,
  },
  title: {
    require: true,
    type: String,
  },
  subtitle: {
    require: true,
    type: String,
  },
  time: {
    require: true,
    type: String,
  },
  date: {
    require: true,
    type: Date,
  },
  avt: {
    type: String,
  },
});
const NotificationAdmin = mongoose.model(
  "notificationadmin",
  notificationAdminSchema
);
module.exports = NotificationAdmin;
